lib/cursor.js

```diff
@@ -992,6 +992,11 @@ var count = function(self, applySkipLimit, opts, callback) {
     command.hint = self.s.cmd.hint;
   }
 
+  // Apply a collation if set
+  if (self.s.cmd.collation) {
```

Comment:
@mbroadst do we need to add a wire version check here?

Reply:
yeah so actually the look here is to add the collation to the `options`, so that the `wireProtocolHandler` can optionally apply it to the command there if collation is supported